CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[7.2.1]](https://github.com/iExecBlockchainComputing/iexec-core/releases/tag/v7.2.1) 2022-12-12
+
+* `sessionId` is now a hash of the public configuration to avoid a reboot of the worker in case the configuration has not changed.
```

Comment:
```suggestion
* Use a `sessionId` which is the hash of the public configuration. A same `sessionId` tells the worker that the configuration has not changed.
```

Reply:
It not really a sessionId after this hotfix. The name is kept to avoid a fix on worker side, but the sessionId name no more exists on scheduler side.

 ```suggestion
* Replace `sessionId` implementation with a hash of the public configuration. From a consumer point of view, a constant hash means that the scheduler configuration has not changed and allows reboot from the scheduler without requiring it from the consumers.
```